docs/src/using-deis/registering-a-user.md

```diff
@@ -72,4 +65,4 @@ If you already have an account, use `deis login` to authenticate against the Dei
     Deis session information is stored in your user's ~/.deis directory.
 
 
-[controller]: ../understanding-deis/components.md#controller
+[router]: https://github.com/deis/router
```

Comment:
We should link to https://github.com/deis/workflow/blob/master/docs/src/understanding-deis/components.md#router as we do elsewhere in the documentation. We can link to deis/router there in a followup so users can find the project, but let's keep the external links to a minimum :)